Include absoluteRuntime in babel preset docs (#5847)
facebook/create-react-app: packages/babel-preset-react-app/README.md

  "presets": [["react-app", { "flow": false, "typescript": true }]]
}
```

## Absolute Runtime Paths

> By default, `babel-preset-react-app` will make `@babel/runtime` imports to use absolute file system paths. To avoid that and use `node_modules/` relative paths set off the `absoluteRuntime` option on `.babelrc`:

Reviewer: Would this be OK? I think it aligns a little better with the rest of this file's options.

```suggestion
Absolute paths are enabled by default for imports. To use relative paths instead, set the `absoluteRuntime` option in `.babelrc` to `false`:
```

```
{
  "presets": [["react-app", { "absoluteRuntime": false }]]

> }

Reviewer: This needs to be closed with ` ``` ` :)
